cdap-app-templates/cdap-etl/cdap-data-pipeline/src/main/java/co/cask/cdap/datapipeline/SmartWorkflow.java

```diff
@@ -177,6 +190,8 @@ public void initialize(WorkflowContext context) throws Exception {
       postActions.put(actionSpec.getName(), (PostAction) context.newPluginInstance(actionSpec.getName(),
                                                                                    macroEvaluator));
     }
+
+    WRAPPERLOGGER.info("Pipeline '{}' running", context.getApplicationSpecification().getName());
```

Comment:
Do you mean?
```
WRAPPERLOGGER.info("Pipeline '{}' {}", context.getApplicationSpecification().getName(), getContext().getState().getStatus().toLowerCase());
```

I think the current form is more readable specially we know that the pipeline will be in RUNNING state in this method, right?